### ISSUE TITLE
Converting multiple AR Max saves of the same game causes the previous save to be overwritten

### ISSUE THREAD

**Issue by PookaMustard** [bucanero/psv-save-converter#2]
If you have multiple saves for the same game (let's say you have four separate GTA San Andreas saves) and you convert all of them at once, only the latest save will be in available, as the first save has been overwritten by the second, then the third, and so on.

This makes it difficult not only to bulk convert saves with scripts, but also a tedious manual process because of all the renaming involved. I suggest at the very least that if a previous PSV (or PSU/MCS) was found with the same filename, that `_1`, `_2`, `_3` and so on are suffixed to the resulting file name.

This issue was found with AR Max saves but I don't have enough evidence to say this also affects other save formats.

**Comment by bucanero** [comment]
Keep in mind that the `.PSV` filename is locked and restricted by the PS3 system, so if the filename doesn't follow the exact ps3 definition the file will be rejected by the PS3 when you try to import it. (no matter if it was properly resigned, hashed, and converted)

You can try yourself, by taking any .PSV file, and rename it for example adding an extra character (or your suggestion `_1`). Then open the PS3, and try to import the PSV using the XMB virtual memory card manager. You'll find that the transfer fails and the PS3 reports an error.

So your issue can't be really solved due to this "mandatory filename" system limitation (or security check).

**Comment by PookaMustard** [comment]
Is it possible to at least warn about overwriting save files? It's probably also possible to put the resulting .PSV in a subfolder with any name while the .PSV itself follows the exact PS3 filename system. If not, offer to rename the .PSV but warn that it needs to be manually renamed later (e.g. save as `.psv.1` but warn that it must be `.psv`).

But at the very least an overwrite warning could do wonders.

**Comment by bucanero** [comment]
> But at the very least an overwrite warning could do wonders.

yes, a check to see if the PSV already exists could be added, so the user decides to overwrite or not.
